idr/connections.py

```diff
@@ -15,55 +14,74 @@ def _configuration_from_url(config_url):
     OMERO binary protocol doesn't support load balancing nor session pinning
     so it has to be done client-side by connecting to a random server/port
     """
+    try:
+        host = re.match('\w+://([^:/]+)', config_url).group(1)
+    except AttributeError:
+        host = config_url
+        config_url = 'https://%s/connection/omero-client.json' % host
```

Comment:
So currently this won't work either either non-`443` ports or with URL prefixes, correct? Probably fine for the moment, but something for us to be aware of.

Reply:
There also me another round of standardization discussions are `connection/omero-client.json` before bringing this back to the mainline. For IDR at the moment, I don't see any blockers.

cc: @will-moore @rgozim